docs/apis/inappbrowser.md

```diff
@@ -0,0 +1,330 @@
+| **`options`** | <code><a href="#systembrowseroptions">SystemBrowserOptions</a></code> | A structure containing some configurations to apply to the System Browser. |
+
+
+#### SystemBrowserOptions
+
+| Prop          | Type                                                                                | Description                              |
+| ------------- | ----------------------------------------------------------------------------------- | ---------------------------------------- |
+| **`android`** | <code><a href="#androidsystembrowseroptions">AndroidSystemBrowserOptions</a></code> | Android-specific System Browser options. |
+| **`iOS`**     | <code><a href="#iossystembrowseroptions">iOSSystemBrowserOptions</a></code>         | iOS-specific System Browser options.     |
+
+
+#### AndroidSystemBrowserOptions
+
+| Prop                      | Type                                                              | Description                                                                                                                      |
+| ------------------------- | ----------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| **`showTitle`**           | <code>boolean</code>                                              | Enables the title display.                                                                                                       |
+| **`hideToolbarOnScroll`** | <code>boolean</code>                                              | Hides the toolbar when scrolling.                                                                                                |
+| **`viewStyle`**           | <code><a href="#androidviewstyle">AndroidViewStyle</a></code>     | Sets the presentation style of CustomTabs.                                                                                       |
+| **`bottomSheetOptions`**  | <code><a href="#androidbottomsheet">AndroidBottomSheet</a></code> | Sets the options for the bottom sheet when this is selected as the viewStyle. If viewStyle is FULL_SCREEN, this will be ignored. |
+| **`startAnimation`**      | <code><a href="#androidanimation">AndroidAnimation</a></code>     | Sets the start animation for when the browser appears.                                                                           |
+| **`exitAnimation`**       | <code><a href="#androidanimation">AndroidAnimation</a></code>     | Sets the exit animation for when the browser disappears.                                                                         |
+
+
+#### AndroidBottomSheet
+
+| Prop          | Type                 | Description                             |
+| ------------- | -------------------- | --------------------------------------- |
+| **`height`**  | <code>number</code>  | Sets the height of the bottom sheet.    |
```

Comment:
Should we state that if no value is passed, then `height` will be 50% of the screen's height?